salt/modules/cassandra_cql.py

```diff
@@ -407,8 +411,13 @@ def cql_query_with_prepare(query, statement_name, statement_arguments, callback_
         salt this-node cassandra_cql.cql_query_with_prepare "name_select" "SELECT * FROM USERS WHERE first_name=?" \
             statement_arguments=['John']
     '''
-    # Backward-compatibility with Python 3.7: "async" is a reserved word
-    asynchronous = kwargs.get('async', False)
+    # Backward-compatibility. This should be removed in Sodium version
+    if 'async' in kwargs:
+        asynchronous = bool(kwargs['async'])
+        msg = 'Use of "async" parameter detected. The "asynchronous" parameter should be used instead.'
```

Comment:
I dont think we should deprecate async, since this is such a small change. So we can ensure users dont need to update their states. Can we remove the deprecation messages?

Reply:
ok, this message removed